Dockerfile

```diff
@@ -99,6 +99,7 @@ COPY apps/correlations apps/correlations
 COPY apps/preferences apps/preferences
 COPY apps/provisioning apps/provisioning
 COPY apps/secret apps/secret
+COPY apps/scope apps/scope
```

Comment:
@macabu what do you think about replacing all of the `COPY apps/XYZ` directive with just `COPY apps apps` instead?